region/norfair/east.json

```diff
@@ -2821,7 +2817,24 @@
           "to": [
             {
               "id": 2,
-              "requires": null,
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"or": [
+                  {"and": [
+                    "Missile",
```

Comment:
Is there anything special about missiles that strong beam combinations don't do?

Reply:
Pretty sure I was just being lazy and didn't want to clutter the code with a bunch of beams but I went back and found you don't need the missiles to achieve that time at all so clutter be gone!